instagram/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Comment = props => {
+    return(
+        <div className='Comment'>
+            <span>{props.comment.username}</span>
+            <span>{props.comment.username}</span>
```

Comment:
Why are you displaying the usernames twice?

Reply:
Will have the extra prop removed.